source/includes/fact-ansi-unicode-driver.rst

```diff
@@ -1,5 +1,6 @@
 .. note::
 
-   ANSI ODBC/Connectors offer maximum performance but have a limited
-   character set. Unicode ODBC/Connectors support a wider character
-   set but may be slightly less performant.
+   The ANSI ODBC driver (``libmdbodbca.so``) offers maximum
+   performance but has a limited character set. The Unicode ODBC
```

Comment:
Why is the limited character set an issue? May help to say that the ANSI driver is more performant _because_  it is limited to the 256 characters in the Latin alphabet. The Unicode driver can support those characters plus the 1.1 million characters in non-Latin languages. It does so with slight performance degradation.

Reply:
I'll add something to the effect that the character set limitation and performance are linked, but I'd rather not go into the details of ANSI vs. Unicode.